### ISSUE TITLE
Can only load and read two pages, "01 introduction" and "01 introduction", in the GitBook version.

### ISSUE THREAD

**Issue by mustig3** [yoshuawuyts/workshop-choo#48]
Can't navigate to other pages, the pages don't want to load.
Reading the tutorial here at github in meantime.

**Comment by yoshuawuyts** [comment]
Yeah gitbook seems to have a fair share of bugs - I think at this point
writing our own might yield the best results
